sdk/javascript/src/symbol/MerkleHashBuilder.js

```diff
@@ -54,4 +55,28 @@ class MerkleHashBuilder {
 	}
 }
 
-module.exports = { MerkleHashBuilder };
+/**
+ * Proves a merkle hash.
+ * @param {Hash256} leafHash Leaf hash to prove.
+ * @param {array<object>} merklePath Merkle hash chain from leaf to root.
```

Comment:
I'm sort of wondering, should we describe in docs how that object actually looks like?